test/reactive-handler.spec.ts

```diff
@@ -662,6 +662,52 @@ describe('ReactiveHandler', () => {
             expect(proxyDesc.get).toEqual(proxyCopiedDesc.get);
             expect(proxyDesc.set).toEqual(proxyCopiedDesc.set);
         });
+        it('should protect against leaking accessors into the blue side', () => {
+            const target = new ReactiveMembrane();
+            const todos = {};
+            let value = 0;
+            let getterThisValue = null;
+            let setterThisValue = null;
+            const newValue = {};
+            function get() {
+                getterThisValue = this;
+                return value;
+            }
+            function set(v) {
+                setterThisValue = this;
+                value = v;
+            }
+            const proxy = target.getProxy(todos);
+            // installing and interacting with proxy
+            Object.defineProperty(proxy, 'entry', {
+                get,
+                set,
+                configurable: true
+            });
+            const proxyDesc = Object.getOwnPropertyDescriptor(proxy, 'entry');
+            expect(proxyDesc.get).toEqual(get);
+            expect(proxyDesc.set).toEqual(set);
+            expect(proxyDesc.get).toEqual(Object.getOwnPropertyDescriptor(proxy, 'entry').get);
+            expect(proxyDesc.set).toEqual(Object.getOwnPropertyDescriptor(proxy, 'entry').set);
```

Comment:
L690-L691 are effectively the same as L687-L689. Unless what you are testing that multiple invocation yields the same result.
```suggestion
            expect(proxyDesc.get).toEqual(get);
            expect(proxyDesc.set).toEqual(set);
```